### ISSUE TITLE
Cannot adjust bottom sheet position when VoiceOver is enabled

### ISSUE THREAD

**Issue by tbesi** [gorhom/react-native-scroll-bottom-sheet#1761]
# Bug

On our iOS app we have a bottom sheet where we display a list of stores above a map screen. We are setting snap points at 30%, 45%, 65% and 95%. The initial snap point is the first one at 30%. While on Android the users can slide the bottom sheet to the desired snap point with the two fingers when using TalkBack, the case is not the same with VoiceOver on iOS. While the bottom sheet has an 'adjustable' role the gestures do not seem to do anything at all.

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet            | ^4.6.1 |
| react-native                    | 0.73.5   |
| react-native-reanimated         | ^3.7.1   |
| react-native-gesture-handler    | ^2.15.0  |

## Steps To Reproduce

On iOS:
1. Have a Bottom Sheet and add some snap points
2. Set the index to 0
3. Enable VoiceOver and set the focus on the Bottom Sheet
4. There is no way to adjust the Bottom Sheet position

Describe what you expected to happen:

1. Should be able to adjust the bottom sheet position

## Reproducible sample code

It can simply be reproduced by using the sample react native app and adding a bottom sheet in the initial screen

```
import BottomSheet from '@gorhom/bottom-sheet';
import React from 'react';
import {SafeAreaView, StatusBar, Text, useColorScheme} from 'react-native';
import {GestureHandlerRootView} from 'react-native-gesture-handler';

import {Colors, Header} from 'react-native/Libraries/NewAppScreen';

function App(): React.JSX.Element {
  const isDarkMode = useColorScheme() === 'dark';

  const backgroundStyle = {
    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
    flex: 1,
  };

  return (
    <GestureHandlerRootView style={{flex: 1}}>
      <SafeAreaView style={backgroundStyle}>
        <StatusBar
          barStyle={isDarkMode ? 'light-content' : 'dark-content'}
          backgroundColor={backgroundStyle.backgroundColor}
        />
        <Header />
        <BottomSheet
          index={0}
          snapPoints={['30%', '45%', '65%', '95%']}
          animateOnMount={true}>
          <Text>Bottom sheet test</Text>
        </BottomSheet>
      </SafeAreaView>
    </GestureHandlerRootView>
  );
}

export default App;
```

**Comment by tbesi** [comment]
This is still an issue. Commenting for activity.

**Comment by virginiab10** [comment]
is still an issue! could be reopened?
